bloxberg/.htaccess

```diff
@@ -2,6 +2,7 @@ Header set Access-Control-Allow-Origin *
 Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
 Options +FollowSymLinks
 RewriteEngine on
+RewriteRule ^schema/research_object_certificate_v1$ https://certify.bloxberg.org/research_object_certificate_v1 [R=301,L,NC]
 RewriteRule ^$ https://www.bloxberg.org/ [R=302,L]
 RewriteRule ^research_object_certificate_v1$ https://certify.bloxberg.org/research_object_certificate_v1 [R=302,L]
-RewriteRule ^(.*)$ https://www.bloxberg.org/$1 [R=302,L]
+RewriteRule ^(.*)$ https://bloxberg.org/ [R=302,L]
```

Comment:
Should the above www.bloxberg.org and this bloxberg.org be synced to be the same?

Reply:
Yes, they should be synced and redirect to the same. We have a redirect on our end from www.bloxberg.org -> bloxberg.org